.github/workflows/build.yml

```diff
@@ -49,7 +49,7 @@ jobs:
     - uses: ./.github/actions/build-setup
       with:
         nim-version: ${{ matrix.nim-version }}
-    - run: nimble test
+    - run: nimble test --warning[BareExcept]:off
```

Comment:
This seems not to be picked up: https://github.com/samdze/playdate-nim/actions/runs/14674842933/job/41189152091?pr=109